pdf/model/font_simple.go

```diff
@@ -132,6 +132,8 @@ func (font pdfFontSimple) GetRuneMetrics(r rune) (fonts.CharMetrics, bool) {
 	return metrics, ok
 }
 
+const defaultFontWidth = 250
```

Comment:
// defaultFontWidth specifies the width used for a glyph if there is nothing else specified.